apps/web/screens/Category/Category.tsx

```diff
@@ -24,29 +24,37 @@ import {
   GET_NAMESPACE_QUERY,
   GET_ARTICLES_IN_CATEGORY_QUERY,
   GET_CATEGORIES_QUERY,
+  GET_LIFE_EVENTS_IN_CATEGORY_QUERY,
 } from '../queries'
 import { CategoryLayout } from '../Layouts/Layouts'
+import { LifeEventCard } from '../../components/LifeEventsCardsSection/components/LifeEventCard'
```

Comment:
Using this component for now, as we have another ticket to fix up the UI based off final designs (that are not complete atm). We will most likely have to create another component that is more specific to Life Events in Categories.